### DISCUSSION TITLE
MPI Parallelism on CPU Nodes

### DISCUSSION THREAD

**Discussion by npiroozan** [deepmodeling/deepmd-kit#1654] [Q&A]
Hello,

Is DeePMD-kit parallelized via MPI on CPU nodes as well as GPU nodes?  Thank you!

**Comment by njzjz** [comment]
Yes.

**Reply by npiroozan** [reply]
Thank you!  Does this apply to training as well?  In my application of DeePMD for the training of Silicon, I saw noticeably reduced performance when implementing MPI on CPU nodes.

**Reply by njzjz** [reply]
MPI training applies data parallelization. The actual effect is to increase the batch size. This is helpful for multiple nodes. But if you only have one node, instead, you should directly increase the batch size.

**Reply by npiroozan** [reply]
Thank you very much.  I had an issue regarding a bottleneck on ProdForceSeA.  Tuning OMP, TF_Intra, and TF_Inter Op settings resolved that.  

As a final question, and please forgive me if this is a foolish question:

Is the current stable release version (2.1.1) an updated version of what was run on Fugaku and Summit in the following paper? :

"Extending the limit of molecular dynamics with ab initial accuracy to 10 billion atoms"

Thank you very much.

**Reply by wanghan-iapcm** [reply]
For summit yes. 

For Fugaku, you can reach the code by the link https://doi.org/10.5281/zenodo.5788368
